movies_test.py

```diff
@@ -0,0 +1,111 @@
+"""Tests the functions of the code.py file"""
+
+import movies
+
+CUSTOMERS = {
+    "Bob" : {"movies":[],
+             "cash" : 100.0},
+    "Jim" : {"movies": ["Xmen 8: The Xmennening"],
+             "cash" : 10.0},
+    "Cary" : {"movies": ["Gigli: The Play: The Book: The movie", "The Bromance"],
+              "cash" : 120.0},
+    "Ricci": {"movies": [],
+              "cash" : 4.0}
+}
+MOVIE_TICKETS = {
```

Comment:
I think I told you to call this MOVIE_TICKETS at one point, but after seeing it again, I think MOVIES is probably a better choice